src/crimson/os/seastore/btree/fixed_kv_btree.h

```diff
@@ -968,7 +968,7 @@ class FixedKVBtree {
           parent.node
         );
         typename internal_node_t::Ref mparent = mut->cast<internal_node_t>();
-        mparent->update(piter, new_addr);
+        mparent->update(mparent->iter_idx(piter.offset), new_addr);
```

Comment:
piter may have been invalid at this point, but it's very strange that it hasn't caused any failure, @athanatos @cyx1231st do you guys have any clue about why?

Reply:
The parent should have been added to the transaction's read-set, right? Why is it possible for the transaction to see an invalidated extent when it is in the read-set? `si_then()` should be able to interrupt the invalidated transaction's future chain before that.

This modification seems identical to the original version, refer to https://github.com/ceph/ceph/blob/f7e122e4b8f671e0555855c0a53d9f748a35dcfb/src/crimson/common/fixed_kv_node_layout.h#L313-L317

Reply:
Ah, yes, I thought `FixedKVInternalNode::update` was `FixedKVNodeLayout::update`. I'm closing this pr.